### ISSUE TITLE
[BUG][JAVA][SERVER] In release 6.2.1, spring server generator creates uncompilable code if some bean properties are marked as required but the setting useBeanValidation is set to false

### ISSUE THREAD

**Issue by raverone** [OpenAPITools/openapi-generator#14293]
##### Description

If I mark some bean properties as required then the library will put ```@NotNull``` annotation above the getter for that property, even if the setting ```useBeanValidation``` is set to false. Also the import statement for the annotation won't be there which makes the code uncompilable.
This is not the case for an older version 6.2.0. In 6.2.0 in such scenario the ```@NotNull``` annotation won't be generated.

##### openapi-generator version

6.2.1

##### OpenAPI declaration file content or url

[Gist](https://gist.github.com/raverone/bcf34804241bb39e40121ef2704ab040)

##### Generation Details

Generation is done with the gradle plugin by executing ```./gradlew :openApiGenerate```

##### Steps to reproduce

I created a minimal project on [github](https://github.com/raverone/open-api-not-null-bug)
Steps to reproduce the bug:
* clone the repo
* cd into the repo directory
* execute ```./gradlew :openApiGenerate```
* examine file build/generated/src/main/java/org/example/bug/http/model/SomeObject.java

##### Expected generated code
```
  @Schema(name = "description", required = true)
  public String getDescription() {
    return description;
  }
```
##### Actual generated code
```
  @NotNull
  @Schema(name = "description", required = true)
  public String getDescription() {
    return description;
  }
```

**Comment by borsch** [comment]
This issue was introduced as part of this issue https://github.com/OpenAPITools/openapi-generator/issues/13365. This change adds `@NotNull` without import statement. That change was part of released in 6.2.1

Later there was a fix to add import statement when `useBeanValidation` is set to `false` - https://github.com/OpenAPITools/openapi-generator/pull/13941. This change haven't been released yet

@raverone could you please try latest snapshot/master build of the generator?

**Comment by raverone** [comment]
Thanks @borsch !
I tried [openapi-generator-cli-6.3.0-20230103.074435-61.jar](https://oss.sonatype.org/content/repositories/snapshots/org/openapitools/openapi-generator-cli/6.3.0-SNAPSHOT/openapi-generator-cli-6.3.0-20230103.074435-61.jar) and the generated code had ```@NotNull``` annotations on required bean fields and correct import statement was included.

**Comment by borsch** [comment]
@raverone I assume this can be closed?

**Comment by raverone** [comment]
@borsch I guess so. Do you know when 6.3.0 will be released? And are you going to include the fix for this  #14295 in 6.3.0?

**Comment by borsch** [comment]
@raverone yep, it would be included. I assume 6.3.0 would be released soon, but I can't tell exact date
